### ISSUE TITLE
Cassandra consistency TWO prevents Kong startup on 0.11.0

### ISSUE THREAD

**Issue by nvx** [Kong/kong#2871]
### Summary

Upgrading a working Kong cluster from 0.10.3 to 0.11.0 fails to start when Cassandra consistency is TWO. Prior to upgrade consistency TWO worked fine.

kong[25128]: Error: /usr/local/share/lua/5.1/kong/cmd/start.lua:28: [cassandra error] could not retrieve current migrations: [cassandra error] [Unavailable exception] Cannot achieve consistency level TWO

Changing consistency level to ONE allows Kong to start correctly. Upgrading from 0.10.3 may not be required to reproduce this bug.

May be related to #1913 

### Steps To Reproduce

Environment: RHEL7 using packages from the Kong RPM repository

1. Setup Kong 0.10.3 with a 3-node Cassandra cluster with the kong keyspace replication set to { 'class' : 'SimpleStrategy', 'replication_factor' : 3 }
2. Upgrade Kong package to 0.11.0
3. Update kong.conf (remove old cluster config, add db_update_propagation)
4. Run Kong migrations up (succeeds)
5. Start Kong

```Error:
/usr/local/share/lua/5.1/kong/cmd/start.lua:62: /usr/local/share/lua/5.1/kong/cmd/start.lua:28: [cassandra error] could not retrieve current migrations: [cassandra error] [Unavailable exception] Cannot achieve consistency level TWO
stack traceback:
        [C]: in function 'error'
        /usr/local/share/lua/5.1/kong/cmd/start.lua:62: in function 'cmd_exec'
        /usr/local/share/lua/5.1/kong/cmd/init.lua:88: in function </usr/local/share/lua/5.1/kong/cmd/init.lua:88>
        [C]: in function 'xpcall'
        /usr/local/share/lua/5.1/kong/cmd/init.lua:88: in function </usr/local/share/lua/5.1/kong/cmd/init.lua:45>
        /usr/local/bin/kong:7: in function 'file_gen'
        init_worker_by_lua:38: in function <init_worker_by_lua:36>
        [C]: in function 'xpcall'
        init_worker_by_lua:45: in function <init_worker_by_lua:43>
```

**Comment by thibaultcha** [comment]
Hi,

Have you tried

**Comment by jgava** [comment]
Tried ONE, TWO and THREE, conclusion just ONE working. same error reported.

**Comment by jgava** [comment]
I guest must be related to cassandra_contact_points

**Comment by thibaultcha** [comment]
Hi,

The consistency setting of Cassandra isn't remotely related to the 
contact points property.

The consistency setting that you chose depends entirely on your 
Cassandra cluster, how you configured it, and the way you want your 
application to behave (availability vs. consistency). I suggest that you 
read some online literature if you wish to use Kong with Cassandra and 
aren't familiar with it, or consider switching to PostgreSQL.

--
Thibault

**Comment by nvx** [comment]
Sorry for the slow reply, I experience the same issue on 0.11.1 as well.

**Comment by thibaultcha** [comment]
This is either a mis-configuration of your cluster, or an issue that we encountered with migrations that has been solved in Kong 0.12/0.13. Closing it and considering it resolved.
